apps/rush-lib/src/logic/PackageJsonUpdater.ts

```diff
@@ -355,18 +355,28 @@ export class PackageJsonUpdater {
       }
       console.log(`Querying NPM registry for latest version of "${packageName}"...`);
 
+      //if the project exists in the local
+      const rushConfigProjects: RushConfigurationProject[] = this._rushConfiguration.projects;
+      rushConfigProjects.forEach(rushConfigProject => {
```

Comment:
Can you use an iterative loop here instead?

```suggestion
      for (const rushConfigProject of rushConfigProjects) {
```